spec/gameSpec.js

```diff
@@ -0,0 +1,91 @@
+// # Scoring
+// * strike - add 10, plus the number of pins knocked down by the next two balls to the score of the previous frame??
+// * spare - add 10, plus the number of pins knocked down by the next ball to the score of the previous frame
+// * regular - add the total of pins knocked down by the two throws to the previous frame.
+
+// knows when all 10 frames are complete and it is the end of the game
+// knows when a perfact game score of 300 is achieved
+
+describe('A throwBalling game', function() {
+	var game;
+
+	beforeEach(function() {
+    game = new Game();
+  });
+
+	describe('frames', function() {
+
+		it('consists of 10 frames', function(){
+			expect(game.frameList.length).toEqual(10);
+		});
+
+		it('ends on frame 10/final frame', function() {
+			expect(game.frameList[9].isFinalFrame).toBe(true);
```

Comment:
Breaking encapsulation & magic number. You might want a function, #getFrame, that returns a frame at a particular index.